client/src/components/StandardFAQ/StandardFAQ.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import { SomeThingsToKeepInMind } from "src/panels/panel_declarations/common_panel_components";
+
+import { KeyConceptList } from "src/components/KeyConceptList/KeyConceptList";
+
+import { secondaryColor } from "src/style_constants/colors.interop.scss";
+
+type StandardFAQProps = {
+  faq_content: React.ReactNode[][]; // TODO: change to [React.ReactNode, React.ReactNode][] when faq_utils gets converted to ts
```

Comment:
The `faq_content` prop could be called `question_answer_pairs` imo, to keep it clearer that it's a straight pass through to the `KeyConceptList` prop of the same nam.